tests/test_core.py

```diff
@@ -736,26 +731,23 @@ def test_get_temp_table_names(engine_testaccount):
     for idx in range(num_of_temp_tables):
         engine_testaccount.execute(
             text(
-                """
-CREATE TEMPORARY TABLE {} (col1 integer, col2 string)
-""".format(
-                    temp_table_name + str(idx)
-                )
+                f"CREATE TEMPORARY TABLE {temp_table_name + str(idx)} (col1 integer, col2 string)"
             ).execution_options(autocommit=True)
         )
     for row in engine_testaccount.execute("SHOW TABLES"):
         print(row)
-    try:
-        inspector = inspect(engine_testaccount)
-        temp_table_names = inspector.get_temp_table_names()
-        assert len(temp_table_names) == num_of_temp_tables
-    finally:
-        pass
+    inspector = inspect(engine_testaccount)
+    temp_table_names = inspector.get_temp_table_names()
+    assert len(temp_table_names) == num_of_temp_tables
 
 
 def test_create_table_with_schema(engine_testaccount, db_parameters):
     metadata = MetaData()
-    new_schema = db_parameters["schema"] + "_NEW"
+    new_schema = (
+        db_parameters["schema"]
+        + "_NEW_"
+        + random_string(5, choices=string.ascii_uppercase)
+    )
```

Comment:
```suggestion
    new_schema = f"{db_parameters["schema"]}_NEW_{random_string(5, choices=string.ascii_uppercase)}"
```
Not sure if this fits in one line